.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
As we're not at the stage of a GUI yet I think we need to adjust these to be more appropriate to the workflow.

At this stage on the development branch I think this just consists of...

```
python topostats/topotracing.py
```

But that's not everything and eventually a bunch of configuration/parameters will be read from a file, so it might be worth asking for the configuration file to be included.

Reply:
I think this is now more appropriate for topostats.

Reply:
I've done this, I hope!